modules/tinymce/src/themes/silver/main/ts/backstage/TooltipsBackstage.ts

```diff
@@ -0,0 +1,63 @@
+import { AlloyComponent, GuiFactory, TooltippingTypes } from '@ephox/alloy';
+import { Fun, Result } from '@ephox/katamari';
+
+export interface TooltipsProvider {
+  readonly getConfig: (spec: { tooltipText: string }) => TooltippingTypes.TooltippingConfigSpec;
+}
+
+export const TooltipsBackstage = (
+  getSink: () => Result<AlloyComponent, any>
+): TooltipsProvider => {
+
+  const tooltipDelay = 800;
+  const intervalDelay = tooltipDelay * 0.2;   // Arbitrary value
```

Comment:
Might do trials and errors to find a delay rate that will give a better UX

Reply:
This will be addressed in an upcoming ticket. TINY-10475